estafette-letsencrypt-certificate/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Kubernetes controller to retrieve and renews tls certificates from Letsencrypt for annotated Kubernetes secrets
+name: estafette-letsencrypt-certificate
+version: 0.1.0
```

Comment:
Is this going to be overwritten when we publish the chart? Also appVersion I guess

Reply:
Yes during the packaging the version and app version are set, see https://github.com/estafette/estafette-extension-helm. During the normal builds it creates a pre-release package, when triggering the release target it creates a final package.